index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/722afd38e8.js" crossorigin="anonymous"></script>
+</head>
+<body>
+   
+  <section class="header">
+    <div class="my-logo"><span>My Logo</span></div>
+
+    <div class="dropdown">
+        <button class="dropbtn"><img src="assets/images/button.png" alt=""></button>
+        <div class="dropdown-content">
+          <a href="#">Portfolio</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </div>
+      </div>
+
+  </section>
+   
+    <section class="headline">
+      
+        <h2>I’m Ahmed Glad to see you!</h2>
+        
+        <p>I’m a software developer! I can help you build a product,
```

Comment:
:red_circle: Kindly make sure you reduce the vertical white space between thee two blocs
|Your Design|Figma Design|
|:----------------:|:----------------:|
|![Screenshot from 2023-01-17 23-52-05](https://user-images.githubusercontent.com/26736582/213023911-f3764992-dd6c-497a-b4e9-99c3d933d1dd.png)|![Screenshot from 2023-01-17 23-53-13](https://user-images.githubusercontent.com/26736582/213023764-a8ccc0ec-cd56-4262-a76e-1970d79d87e4.png)
|

Reply:
resolved :+1:

Reply:
- [ ] Also, kindly make sure you stick to the designs much as possible. The typos (fonts-size, family, wight) should be similar to the provided template.